packages/ui/src/modal/ConfirmationModal.tsx

```diff
@@ -13,21 +13,23 @@ interface ConfirmationModalProps {
     isModalOpen: boolean;
     onRequestClose: () => void;
     action: () => void;
+    actionButtonLabel?: string | ReactNode;
```

Comment:
I can see this added in this PR and a node/element is passed in. Do we need to have a string here?

Reply:
Updated https://github.com/bigcommerce/checkout-js/pull/2106/commits/a0a88140043af8bfc005341e58d86ef560f1d227